### ISSUE TITLE
 The program icon cannot be loaded.

### ISSUE THREAD

**Issue by iDTer** [typora/typora-issues#3700]
Hey, guys! I have a little problem. I like this software very much, but when I installed typora after changing the computer, the icon cannot be loaded. I have tried to uninstall and install this program repeatedly, or use the old version, but it is useless. I also tried to google some mesures, but still the same. I guess it may be caused by **high DPI**. I would like to ask if you have encountered a similar problem? (Run screenshots are posted below.
Anyway, thx!

machine: Thinkpad x1c 
OS: win10
version: 0.9.92(beta)

![2020-07-14_141221](https://user-images.githubusercontent.com/30014931/87395750-6ec76c80-c5e4-11ea-861f-8b0ba743a441.png)

Picture of the taskbar: 
![2020-07-14_141339](https://user-images.githubusercontent.com/30014931/87395768-738c2080-c5e4-11ea-9520-707bf55d8a23.png)

**Comment by abnerlee** [comment]
Does the icon of exe file shows in Explorer, does icon shows in taskbar?

What is your DPI and try launch in compatibility mode, will the icons shows?

And do you have a anti-virus software running and maybe deleting files when installing Typora?

**Comment by iDTer** [comment]
> Does the icon of exe file shows in Explorer, does icon shows in taskbar?
> 
> What is your DPI and try launch in compatibility mode, will the icons shows?
> 
> And do you have a anti-virus software running and maybe deleting files when installing Typora?

1. Yes, the icon of exe file can shows in Explorer and also can shows in taskbar.

2. The display resolution is 3840x2160. I tried to launch it in compatibility mode, but the icon not shows.

3. I checked the history of Windows defence and found no file deletion behavior when installing Typora.

I find that it looks more comfortable in the Unibody mode, because there is no need to load icons at this time. Anyway, thx for your reply!

![post](https://user-images.githubusercontent.com/30014931/88369319-50baf280-cdc2-11ea-8de8-60abd2fb76e5.png)
